### PR TITLE
How to submit to challenge

### PR DESCRIPTION
**Pull request by thomasyu888**

### CODE REVIEW

**Comment by thomasyu888** [comment]
Add more information on merge wiki script

**Comment by thomasyu888** [comment]
Lets review this sometime

**Comment by thomasyu888** [comment]
Can we review this?

**Comment by kdaily** [comment]
Sure, request a review, from me if you wish! :-D

File: articles/challenge_participation.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+title: Participating in a Challenge
+layout: article
+excerpt: Learn how to participate in challenges and submit to evaluation queues.
```
Comment (kdaily) [inline articles/challenge_participation.md]: why specifics on submitting to eval queues in the description? doesn't seem to be required terminology to challenge participation.

File: articles/challenge_participation.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+<style>
+#image {
+    width: 100%;
+}
+#toobig {
+    width: 45%;
+}
+</style>
+
+This tutorial will teach you all the steps of participating in a challenge.
```
Comment (kdaily) [inline articles/challenge_participation.md]: remove "all" or add "the"

File: articles/challenge_participation.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## Challenge Registration
+Participants **must** be registered for the challenge if they want to submit and participate.  The registration button can be found on the home page or `How to Participate` page for every challenge.  *In order to be fully registered for any challenge, you must (1) have a Synapse account and become a [certified user](http://docs.synapse.org/articles/getting_started.html#becoming-a-certified-user); (2) agree to the DREAM Rules of participation, and (3) agree to the Terms of Use to work with the Challenge data.*
```
Comment (kdaily) [inline articles/challenge_participation.md]: Describe what and how before denoting that it's a hard requirement.

File: articles/challenge_participation.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+
+## Join or Create a Team
+Although not mandatory, we encourage you to be part of a team. You can either join a team or create your own team of collaborators. See instructions on how to form your own team of collaborators [here](http://docs.synapse.org/articles/teams.html).  It is important to note that you **cannot** be on more than one team.  Once you have submitted as a team or individual, you cannot submit as another team.  After you form a team, you must register your team to the challenge - there will be a place to do this in every challenge wiki.
```
Comment (kdaily) [inline articles/challenge_participation.md]: Remove 'although not mandatory" - 'encourage' says the same thing.

Comment (kdaily) [inline articles/challenge_participation.md]: Confusing about who 'you' is when describing submitting as a team.

File: articles/challenge_participation.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+## Accessing Challenge Data
+The data stored in the Synapse data repository can be accessed using the Synapse website or programmatically using the Synapse R or Python clients. Instructions on using Synapse are provided in the [Synapse User Guide](http://docs.synapse.org/).  File descriptions are provided on the Data Description page in each challenge wiki.  See instructions
```
Comment (kdaily) [inline articles/challenge_participation.md]: Synapse is not a data repository

Comment (kdaily) [inline articles/challenge_participation.md]: Link to specific pages for data access.

Comment (kdaily) [inline articles/challenge_participation.md]: Last statement not a sentence

File: articles/challenge_participation.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+## Accessing Challenge Data
+The data stored in the Synapse data repository can be accessed using the Synapse website or programmatically using the Synapse R or Python clients. Instructions on using Synapse are provided in the [Synapse User Guide](http://docs.synapse.org/).  File descriptions are provided on the Data Description page in each challenge wiki.  See instructions 
+
+
+## Run your Algorithms and Submit to the Challenge
+
+There are multiple ways a participant can submit to a challenge queue by using the R, python or web client.  The most conventional way is to submit through the web.  All submissions must be uploaded onto a private synapse page.  Follow instructions [here](http://docs.synapse.org/articles/getting_started.html#project-and-data-management-on-synapse) on how to upload to a project. Most challenge queues will be labeled by `challengename-subchallenge#` as a challenge may have different questions that it may want participants to answer.
```
Comment (kdaily) [inline articles/challenge_participation.md]: Voice changed - started active, now passive.

File: articles/challenge_participation.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+After doing so, pick the challenge you want to submit to, in this case (My Example Challenge). Click Next and follow the steps to complete your submission.
+
+<img id="toobig" src="/assets/images/submitToChallenge.png">
+	{% endtab %}
+
+{% endtabs %}
+
+## Share Ideas and Ask Questions
+
+Every challenge has a challenge forum for participants (the 'Discussion' tab on the Challenge's project).  The forum is a space for participants to ask any questions and raise ideas.
```
Comment (kdaily) [inline articles/challenge_participation.md]: Change 'challenge forum' to 'discussion forum'

Comment (kdaily) [inline articles/challenge_participation.md]: 'on the Challenge Project page'

File: articles/challenge_participation.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+<img id="toobig" src="/assets/images/submitToChallenge.png">
+	{% endtab %}
+
+{% endtabs %}
+
+## Share Ideas and Ask Questions
+
+Every challenge has a challenge forum for participants (the 'Discussion' tab on the Challenge's project).  The forum is a space for participants to ask any questions and raise ideas.  
+
+Instructions on how to use the challenge forum can be found [here](http://docs.synapse.org/articles/discussion.html)
```
Comment (kdaily) [inline articles/challenge_participation.md]: discussion forum

**Comment by thomasyu888** [comment]
Thanks Kenny! will update this soon.

File: articles/challenge_participation.md (hunk 5 above)
Comment (thomasyu888) [inline articles/challenge_participation.md]: Should I provide example pages? Each challenge has different data access restrictions

**Pull request merged**